app/src/components/submissionPage/SubmissionTextBox.tsx

```diff
@@ -52,37 +65,83 @@ const SubmissionTextBox: React.FunctionComponent = (props: Props) => {
     }
 
     /* Updates the value in the webhook "message" */
-    function handleChange(event, newValue, newPlainTextValue, mentions) {
+    function handleMessageTextChanged(
+        event,
+        newValue,
+        newPlainTextValue,
+        mentions
+    ) {
         setMessage(event.target.value);
         setConfirmation(false);
         setError("");
         setSnappedUsers(mentions);
     }
 
     return (
-        <div className="SubmissionTextBox">
-            <form onSubmit={handleSubmit}>
-                <h1>Submit Snap</h1>
-                <MentionsInput
-                    style={{ zIndex: 1 }}
-                    value={message}
-                    onChange={handleChange}
-                    maxLength={GetExtraLength(snappedUsers)}
-                >
-                    <Mention
-                        style={{ backgroundColor: "#daf4fa", zIndex: 0 }}
-                        trigger="@"
-                        data={props.snappables}
-                    />
-                </MentionsInput>
-                <input type="submit" value="Submit" />
-            </form>
-            <CharactersLeftDisplay
-                snappedUsers={snappedUsers}
-                message={message}
-            />
-            <OnSubmitMessageDisplay confirmation={confirmation} />
-            <SubmissionBoxErrorDisplay error={error} />
+        <div className="container-sm">
+            <div className="row justify-content-md-center">
+                <LeftSideOfScreen className="col-sm col-lg-5">
+                    <SnapCupText>
+                        Add a Snap to the current SnapCup.
+                    </SnapCupText>
+                    <ElleImg src={Elle} className="w-100" />
+                </LeftSideOfScreen>
+                <RightSideOfScreen className="col-sm col-lg-5">
```

Comment:
It doesn't appear to be having a significant impact but `col-sm` means that at the smallest resolutions this isn't actually a column at all. Chaning `col-sm` to `col` would be safer.